worker/deploy/ansible-playbook/deploy-worker.yml

```diff
@@ -0,0 +1,12 @@
+---
+- hosts: worker-machine
+  vars:
+    setup: bash -x setup.sh
+  tasks:
+    - name: "copy setup file"
+      copy: src="../setup.sh" dest="/home/ubuntu"
+    - name: Setting up environment
+      shell: "{{ setup }} {{ nfs_server }} {{ nfs_server_dir }}"
```

Comment:
where these nfs variables have been set?

Reply:
These variables are set in the `ansible-playbook` call from `install.sh` script.